pkg/resource/resource.go

```diff
@@ -159,34 +160,58 @@ func (c *creator) CreateFromYAML(
 	return nil
 }
 
+func (r *resource) GetObjectsFromYAML(ctx context.Context, yamlFile []byte) (*unstructured.UnstructuredList, error) {
```

Comment:
I don't see `ctx` being used in the function.
```suggestion
func (r *resource) GetObjectsFromYAML(yamlFile []byte) (*unstructured.UnstructuredList, error) {
```

Reply:
fixed